xonsh/environ.py

```diff
@@ -865,6 +865,15 @@ class GeneralSetting(Xettings):
         "If enabled, the CommandsCache is saved between runs and can reduce the startup time.",
     )
 
+    COMMANDS_CACHE_IGNORE_MTIME_CHECK = Var.with_default(
+        True,
+        "If enabled, ignore mtime check when checking if directories changed"
+        "will decrease performance, but ensures scripts will run as expected."
```

Comment:
reformat. it ensures executables' path will be resolved to correct one on platforms where the directories `mtime` field isn't represented accurate enough (nanoseconds on Darwin, but seconds on Linux).